warehouse/query-core/src/test/java/datawave/query/jexl/visitors/UniqueExpressionTermsVisitorTest.java

```diff
@@ -236,21 +239,31 @@ public void testDuplicateSubTreeDisjunction() throws ParseException {
     
     private void visitAndValidate(String original, String expected) throws ParseException {
         ASTJexlScript originalScript = JexlASTHelper.parseJexlQuery(original);
-        ASTJexlScript expectedScript = JexlASTHelper.parseJexlQuery(expected);
-        
-        // Flatten scripts
-        originalScript = TreeFlatteningRebuildingVisitor.flatten(originalScript);
-        expectedScript = TreeFlatteningRebuildingVisitor.flatten(expectedScript);
-        
-        // Assert that both original and expected query strings parse as expected
-        assertEquals("Original query did not parse cleanly" + original, original, JexlStringBuildingVisitor.buildQuery(originalScript));
-        assertEquals("Expected query did not parse cleanly" + expected, expected, JexlStringBuildingVisitor.buildQuery(expectedScript));
         
         // Remove duplicate terms from within expressions.
         ASTJexlScript visitedScript = UniqueExpressionTermsVisitor.enforce(originalScript);
         
-        // De-duped query should match expected
-        assertEquals(expected, JexlStringBuildingVisitor.buildQuery(visitedScript));
```

Comment:
This check should be left in. There have been cases were the tree equality visitor said trees were equal but the queries parsed into different strings.

Reply:
No problem, it's been added back in.